src/components/duelling-picklist/duelling-picklist.style.js

```diff
@@ -30,6 +40,54 @@ const StyledDuellingPicklist = styled.div`
   ${StyledPicklist}:last-of-type {
     padding-left: 36px;
   }
+  ${StyledDraggableContainer} {
```

Comment:
Are these styles really needed? They're already defined in `Picklist` component so it's kinda redundant

Reply:
These are needed to match style for picklist-only.

Reply:
I understand that, but we'll end up with two sets of styles that do exactly the same. It is very likely that someone doing changes to this component in the future will not be aware that the styling changes related to the animation or the scrollbar have to be applied in two different places. Please try to play around with the component structure so that it's not necessary to add these styles at all.